affirm/src/main/java/com/affirm/android/CardRequest.java

```diff
@@ -0,0 +1,160 @@
+package com.affirm.android;
+
+import android.os.Handler;
+import android.os.Looper;
+
+import androidx.annotation.NonNull;
+
+import com.affirm.android.exception.APIException;
+import com.affirm.android.exception.AffirmException;
+import com.affirm.android.exception.ConnectionException;
+import com.affirm.android.model.CardCancelResponse;
+import com.affirm.android.model.CardDetails;
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.io.IOException;
+
+import okhttp3.Call;
+import okhttp3.Callback;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+
+import static com.affirm.android.AffirmConstants.CANCEL_CHECKOUT_CARD_PATH;
+import static com.affirm.android.AffirmConstants.GET_CHECKOUT_CARD_PATH;
+import static com.affirm.android.AffirmTracker.TrackingEvent.NETWORK_ERROR;
+import static com.affirm.android.AffirmTracker.TrackingLevel.ERROR;
+import static com.affirm.android.AffirmTracker.createTrackingNetworkJsonObj;
+
+public class CardRequest {
```

Comment:
I imagine this should implement `AffirmRequest` like the other Requests

Reply:
Fixed